common/src/langs.rs

```diff
@@ -14,6 +14,7 @@ pub struct Lang {
     pub latest_version: &'static str,
     pub icon: &'static str,
     pub extra_mounts: &'static [(&'static str, &'static str)],
+    pub extension: &'static str,
```

Comment:
I would prefer giving extra time to kotlin and vyxal specifically rather than every lang. Maybe add a time bonus field?